includes/reports/overview/list-table/class-sensei-reports-overview-list-table-abstract.php

```diff
@@ -342,10 +344,15 @@ class="sensei-date-picker"
 		 * @param {string} $report_type The report type.
 		 */
 		do_action( 'sensei_reports_overview_after_top_filters', $this->type );
-		?>
 
-		<?php
-		submit_button( __( 'Filter', 'sensei-lms' ), '', '', false );
+		$filters_content = ob_get_clean();
+
+		if ( ! empty( trim( $filters_content ) ) ) {
```

Comment:
An ugly solution to not display the "Filter" button when there's no filter. I couldn't think of another solution without a big change. Accepting suggestions. :)